nikola/shortcodes.py

```diff
@@ -209,14 +209,58 @@ def _parse_shortcode_args(data, start, shortcode_name, start_pos):
     raise ParsingError("Shortcode '{0}' starting at {1} is not terminated correctly with '%}}}}'!".format(shortcode_name, _format_position(data, start_pos)))
 
 
+def _extract_shortcodes(data):
```

Comment:
Why use a private function name? Isn't this function supposed to be used by plugins? It shouldn't start with an underscore IMO.